subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/accessors/AccessorFragments.kt

```diff
@@ -824,3 +822,41 @@ fun KotlinClassMetadata.FileFacade.Writer.writeElementAccessorMetadataFor(
 private
 fun accessorDescriptorFor(receiverType: InternalName, returnType: InternalName) =
     "(L$receiverType;)L$returnType;"
+
+
+private
+fun ConfigurationEntry<AccessorNameSpec>.getDeclarationDeprecationMessage() = if (hasDeclarationDeprecations()) {
+    val deprecationType = SingleMessageLogger.ConfigurationDeprecationType.DEPENDENCY_DECLARATION
+    val summary = "The ${target.original} configuration has been deprecated for ${deprecationType.displayName()}."
+    val suggestion = "Please ${deprecationType.usage} the ${dependencyDeclarationAlternatives.map(::quote).joinToString(" or ")} configuration instead."
+    "$summary $suggestion"
+} else {
+    ""
+}
+
+private
+fun ConfigurationEntry<AccessorNameSpec>.getDeclarationDeprecationBlock() = if (hasDeclarationDeprecations()) {
+    "\n                    @Deprecated(message = \"${getDeclarationDeprecationMessage()}\")"
+} else {
+    ""
+}
+
+
+private
+fun quote(str: String) = "'$str'"
+
+
+private
+fun BytecodeFragmentScope.publicStaticMaybeDeprecatedMethod(jvmMethodSignature: JvmMethodSignature,
```

Comment:
💄 Our Kotlin convention is to put each parameter on its own line, IDEA has a shortcut for that, _Put parameters on separate lines_. You can fix the declaration by first running _Put parameters on one line_ on it.

Reply:
Okay, but it seems there are lots of places where this convention is not applied :)